plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/lsp/AmazonQLanguageClientImpl.kt

```diff
@@ -413,6 +414,16 @@ class AmazonQLanguageClientImpl(private val project: Project) : AmazonQLanguageC
         return refreshVfs(params.newPath)
     }
 
+    override fun sendChatOptionsUpdate(params: LSPAny) {
```

Comment:
Will this actually trigger the new code path

Reply:
One alternative is just to suppress the warning

Reply:
it sends an empty payload so the seems pointless but safe 
```
2025-06-02 16:46:36,111 [ 181028]   INFO - software.aws.toolkits.jetbrains.services.amazonq.lsp.AmazonQServerInstance - [Trace - 04:46:36 PM] Received notification 'aws/chat/chatOptionsUpdate'
Params: {
  "tabId": "c70vdc"
}

```